x509util/certcheck/certcheck.go

```diff
@@ -59,38 +59,31 @@ func main() {
 
 	failed := false
 	for _, filename := range flag.Args() {
-		dataList, err := x509util.ReadPossiblePEMFile(filename, "CERTIFICATE")
+		chain, err := chainFromFile(filename)
 		if err != nil {
-			glog.Errorf("%s: Failed to read data: %v\n", filename, err)
+			glog.Errorf("%v\n", err)
```

Comment:
[preexisting] I guess the `Errorf` calls through-out don't need the trailing `\n`